packages/docusaurus/src/server/__tests__/__snapshots__/routes.test.ts.snap

```diff
@@ -159,10 +126,7 @@ export default [
 exports[`loadRoutes loads route config with empty (but valid) path string 1`] = `
 {
   "registry": {
-    "__comp---hello-world-jse-0-f-b6c": {
-      "loader": "() => import(/* webpackChunkName: '__comp---hello-world-jse-0-f-b6c' */ 'hello/world.js')",
-      "modulePath": "hello/world.js",
-    },
+    "__comp---hello-world-jse-0-f-b6c": "hello/world.js",
```

Comment:
why do these snapshots change if behavior is the same as before? 🤷‍♂️ 

The new snapshot format is not even looking like what I have running locally, so I don't understand what is happening here

If it's a refactor, snapshots shouldn't change? 🤷‍♂️

Reply:
This snapshot is a shot of the internal return of `loadRoutes`, not what `registry.js` looks like. We have another snapshot for `registry.js` which doesn't change at all.

Reply:
I see thanks 👍